### ISSUE TITLE
[FEATURE]: missed version & help flags output for `victoriametrics_backend_plugin_linux_amd64`

### ISSUE THREAD

**Issue by denisgolius** [VictoriaMetrics/grafana-datasource#68]
```posh
./victoriametrics_backend_plugin_linux_amd64 -h
{"@level":"info","@message":"Starting VictoriaMetrics datasource backend ...","@timestamp":"2023-04-04T11:54:11.508891+03:00"}
{"@level":"info","@message":"Profiler","@timestamp":"2023-04-04T11:54:11.509006+03:00","enabled":false}
Usage of ./victoriametrics_backend_plugin_linux_amd64:
  -standalone
    	should this run standalone
    	
./victoriametrics_backend_plugin_linux_amd64 -v
{"@level":"info","@message":"Starting VictoriaMetrics datasource backend ...","@timestamp":"2023-04-04T11:56:56.782846+03:00"}
{"@level":"info","@message":"Profiler","@timestamp":"2023-04-04T11:56:56.782923+03:00","enabled":false}
flag provided but not defined: -v
Usage of ./victoriametrics_backend_plugin_linux_amd64:
  -standalone
    	should this run standalone

 ./victoriametrics_backend_plugin_linux_amd64 -version
{"@level":"info","@message":"Starting VictoriaMetrics datasource backend ...","@timestamp":"2023-04-04T11:57:04.635319+03:00"}
{"@level":"info","@message":"Profiler","@timestamp":"2023-04-04T11:57:04.635397+03:00","enabled":false}
flag provided but not defined: -version
Usage of ./victoriametrics_backend_plugin_linux_amd64:
  -standalone
    	should this run standalone

./victoriametrics_backend_plugin_linux_amd64 --version
{"@level":"info","@message":"Starting VictoriaMetrics datasource backend ...","@timestamp":"2023-04-04T11:57:06.658022+03:00"}
{"@level":"info","@message":"Profiler","@timestamp":"2023-04-04T11:57:06.658156+03:00","enabled":false}
flag provided but not defined: -version
Usage of ./victoriametrics_backend_plugin_linux_amd64:
  -standalone
    	should this run standalone
```

Would be great to have this flags output which help to recognized the version of executable file and how to use them as well.

**Comment by hagen1778** [comment]
@denisgolius this was addressed in https://github.com/VictoriaMetrics/grafana-datasource/pull/104 and available starting from the next release.

**Comment by dmitryk-dk** [comment]
The issue was implemented and included in the [v0.5.0](https://github.com/VictoriaMetrics/grafana-datasource/releases/tag/v0.5.0) release
